notebooks/road_following/data_collection_gamepad.ipynb

```diff
@@ -125,8 +125,11 @@
    "source": [
     "camera = Camera()\n",
     "\n",
-    "image_widget = widgets.Image(format='jpeg', width=224, height=224)\n",
-    "target_widget = widgets.Image(format='jpeg', width=224, height=224)\n",
+    "widget_width = 224\n",
```

Comment:
I'd suggest using ``camera.width`` and ``camera.height`` similar to the other notebook.  This would be directly in place of ``widget_width`` and ``widget_height``.  This will keep the proper association.  The  width / height of widgets corresponds to browser side rendering and can differ from the camera image shape.